pennylane/ops/channel.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in noisy
+quantum channels supported by PennyLane, as well as their conventions.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access
+
+import numpy as np
+
+from pennylane.operation import Channel
+
+
+class AmplitudeDamping(Channel):
+    r"""AmplitudeDamping(gamma, wires)
+    Amplitude damping channel in the Kraus representation.
```

Comment:
For all channels, I would remove 'in the Kraus representation'. This is an implementation detail; a device doesn't _have_ to use the Kraus matrices when implementing a channel. Instead, I would make the summary line a bit more informative, e.g.,

```
Single-qubit amplitude damping error channel.